aten/src/ATen/core/jit_type.h

```diff
@@ -1638,7 +1638,8 @@ inline at::ScalarType scalarTypeFromJitType(const c10::TypePtr& type) {
 // input because of a lack of operator support for NumberType
 CAFFE2_API c10::optional<TypePtr> unifyTypes(
     const TypePtr& t1,
-    const TypePtr& t2);
+    const TypePtr& t2,
+    bool default_to_any = false);
```

Comment:
default to any is somewhat self-explanatory but maybe worth revising the comments above